specification/ai/Azure.AI.Unified/connections/models.tsp

```diff
@@ -0,0 +1,72 @@
+import "@typespec/rest";
+import "@azure-tools/typespec-autorest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@typespec/openapi";
+import "@typespec/versioning";
+import "../credentials/models.tsp";
+
+using TypeSpec.OpenAPI;
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+
+namespace Azure.AI.Unified;
+
+@doc("Response from the listSecrets operation")
+model Connection {
+  @doc("A unique identifier for the connection")
+  @visibility("read")
+  id: string;
+
+  @doc("The name of the resource")
+  @visibility("read")
+  name: string;
+
+  @doc("Credential used to connect to the external resource.")
+  @visibility("read")
+  credentials: BaseCredential;
+
+  @doc("Category of the connection")
+  @visibility("read")
+  category: ConnectionType;
```

Comment:
We have 'category` of type `ConnectionType`.... wouldn't it be better to have `ConnectionCategory`? or `type: ConnectionType`?

Reply:
renamed to `ConnectionCategory`